packages/api/lib/services/users.ts

```diff
@@ -54,3 +57,26 @@ export async function createUser(user: RegisterUser) {
       return e
     })
 }
+
+export async function updateLanguage(
+  id: String,
+  language: Languages
+): Promise<any> {
```

Comment:
Wonder if we can replace the `any` here too! What does `db.query` return?

Reply:
Hard to say, it can be one of several objects and it also depends on what we set as the returning fields on that table row.